tools/rosbag/src/record.cpp

```diff
@@ -299,6 +313,7 @@ int main(int argc, char** argv) {
     // Run the recorder
     rosbag::Recorder recorder(opts);
     int result = recorder.run();
-    
+    ros::waitForShutdown();
```

Comment:
Why is this necessary? The `run()` method on the `recorder` already loops until a shutdown is triggered.

Reply:
My mistake, will remove.